Source/TailwindTraders.ShippingManagement/Services/ResponseService.cs

```diff
@@ -78,9 +78,10 @@ private void ParseHeader(RecognizedForm page)
         {
             foreach (var header in _wrap.Keys)
             {
-                if (page.Fields.Any(c => c.Key[header.KeyPos].ToString() == header.Key))
```

Comment:
`page.Fields` is a dictionary, so instead of all of this, you could just do:
```
if (page.Fields.TryGetValue(header.Key, out FormField kv))
{
    switch(header.type) ...
}
```

Reply:
@maririos - I try to run your code but was not successful, because this code get `key` value. And the below code I had updated was runnable. 
```
if (page.Fields.Any(c => c.Value.LabelData.Text == header.Key))
{
   var kv = page.Fields.Where(c => c.Value.LabelData.Text == header.Key).First();
```